Updated docs and scripts

Parth4337 opened the pull request:
Updated scripts in deploy/testnet and changed default/home folder from $HOME to $HOME/.onomy
Changed docs/instructions accordingly put everything in $HOME/.onomy
All the binaries are now placed in $HOME/.onomy/bin 

Merged the 'setting-up-a-validator' and 'setting-up-a-validator-manual' as well as 'setting-up-afull-node' and 'setting-up-a-full-node-manual. added sections for both manual and scripted installations in same file.

Added RPM file to deploy/testnet and added instructions to install that file in pre-installation doc

Commit:
deploy->testnet->bin.sh
Condensed dnf statement(s) and removed 'screen' package. Added commented out code for adding EPEL Repo and Enabling code-ready builder, in order to install 'moreutils' package

Commit:
deploy->testnet->bin.sh moved installation of nodejs from middle of the file to top, with other packages

Parth4337 committed:
deploy->testnet->bin.bash Removed redundunt variable $HOME_DIRECTORY, replaced with $HOME. Exported all the path variables to ~/.bashrc to make changes to $PATH consistent

Parth4337 committed:
deploy/testnet/bin.sh fixed typo /.bashrc_ -> /.bashrc
docs/testnet/onomy-testnet-docs/faucet.md improved docs

Parth4337 committed:
Merge pull request #1 from Parth4337/testnet-docs

Testnet docs

Parth4337 committed:
Removed facet.md file

Parth4337 committed:
t push origin devMerge branch 'onomyprotocol-dev' into dev
resolved coflict between onomy/dev branch and local dev branch

Parth4337 committed:
added faucet.md file

Parth4337 committed:
docs/testnet/onomy-testnet-docs/faucet.md changed a bit of technical details
docs/testnet/onomy-testnet-docs/pre-installation.md improved docs

Parth4337 committed:
docs/testnet/onomy-testnet-docs/pre-installation.md separated the file in two sections to highlight two different ways to install binaries

Parth4337 committed:
Changes in bin.sh based on PR Review, changed branch of pulled git repo from dev to v0.0.1

Parth4337 committed:
Merged docs for setting up a full node under setting-up-fullnode.md
Changed default path of test-chain data from /home/parth/ to /home/parth/.onomy/
Changed docs to reflect same

Parth4337 committed:
Removed accidently added swp file

Parth4337 committed:
Made some technical and formatiing changes to docs/testnet/onomy-tesetnet-docs/setting-up-a-full-node.md

Parth4337 committed:
Removed docs/testnet/onomy-testnet-docs/setting-up-a-full-node-manual.md:

Parth4337 committed:
fixed typos grammar and formatting

Parth4337 committed:
docs/testnet/onomy-testnet-docs/setting-up-a-fullnode Fixed bullets not indented

Parth4337 committed:
docs/testnet/onomy-testnet-docs/setting-up-a-validator.md improved docs
docs/testnet/onomy-testnet-dpcs/setting-up-a-full-node.md fixed typos and formatting errors

Parth4337 committed:
Merge branch 'onomyprotocol:dev' into dev

Parth4337 committed:
docs/testnet/onomy-testnet-docs/setting-up-a-validator.md merged both manual and scripted versions. improved technical details and formatting
deplot/testnet/peer-validator changed $CURRENT_WORKING_DIR variable to $ONOMY_HOME. Changed default path from $HOME/* to $HOME/.onomy/*"

Parth4337 committed:
docs/testnet/onomy-testnet-docs/setting-up-a-validator.md Fixed typos and grammar

Parth4337 committed:
Fixed typos and grammar

Parth4337 committed:
Merge branch 'dev' of https://github.com/parth4337/onomy into dev

Parth4337 committed:
Added onomy-0.1.0-1.x86_64.rpm to deploy/testnet
Added instructions for installing rpm in Pre-Installation.md file
changed default home directory of all onomy artifects and files to $HOME/.onomy and updated documents accordingly

dzmitryhil commented on docs/testnet/onomy-testnet-docs/setting-up-a-validator.md:
````diff
@@ -1,132 +1,201 @@
-Note: 1. Script will ask for enter faucet url get faucet token(Please enter http://testnet1.onomy.io:8000/)
 ```
+
+Script will ask for faucet url to get faucet token (Please enter http://testnet1.onomy.io:8000/)
````
can we set the default value "http://testnet1.onomy.io:8000/"  ?

Parth4337 replied:
Done

dzmitryhil commented on docs/testnet/onomy-testnet-docs/setting-up-a-validator.md:
````diff
@@ -1,132 +1,201 @@
+
 You can check the validators of the Onomy chain by running:
 ```
 curl http//localhost:26657/validators
````
BTW, The same info you can get using cli:  "onomyd q staking validators"

dzmitryhil commented on deploy/testnet/peer-validator/makeValidator.sh:
```diff
@@ -32,7 +32,7 @@ GRAVITY_VALIDATOR_NAME=$(jq .validator_name $HOME/val_info.json | sed 's#\"##g')
 STAKE_DENOM="nom"
 #NORMAL_DENOM="samoleans"
```
I guess we don't need this line

Parth4337 replied:
Removed

Parth4337 committed:
deploy/testnet/peer-validator/make-validator.sh Added default value of faucet URL while reading, removed a couple of unneeded comments
deploy/testnet/*.sh changed echo-read pair in all the scripts to 'read -p'
docs/testnet/onomy-testnet-docs/setting-up-a-validator.md added alternate command to check info of validators

Parth4337 committed:
deploy/testnet/peer-validators Changed file names to init-full-node.sh and init-validator.sh. Made necessary changes in docs

Changed version number of RPM from 0.1.0 to 0.0.1
Resolved the comments raised in PR #29

Parth4337 committed:
docs/testnet/onomy-testnet-docs/setting-up-a-validator fixed a typo onomy->onomyd

Parth4337 committed:
deploy/testnet/bin.sh fixed onomyd module not installing in bin.sh
